Python/pywarpx/picmi.py

```diff
@@ -1145,16 +1145,11 @@ def initialize_warpx(self, mpi_comm=None):
             return
 
         self.warpx_initialized = True
-        pywarpx.warpx.init(mpi_comm)
+        pywarpx.warpx.init(mpi_comm, max_step=self.max_steps, stop_time=self.max_time)
```

Comment:
Let's not add this in the `init` method but as regular inputs parameters via ParamParse?

Reply:
These are the two parameters that WarpX uses to stop the simulation. They are passed in here and then added to to the list of arguments that are passed to the call to `amrex_init` which adds them to the ParmParse database of parameters. If a Python user has some other means for stopping the simulation, they would leave the max_steps and max_time unset, in which case these parameters would not be added to the argument list.